answerking_app/views/order_views.py

```diff
@@ -180,36 +180,21 @@ def put(
                 error_msg,
                 status=404,
             )
-
         try:
-            body: dict | None = json.loads(request.body)
+            body = json.loads(request.body)
             if not body:
                 return HttpResponse(status=400)
-            quantity: int = body["quantity"]
-        except KeyError as e:
-            error_msg: ErrorMessage = {
-                "error": {
-                    "message": "Failed data validation",
-                    "details": f"'quantity' not found in json body.",
-                }
-            }
-            return JsonResponse(
-                error_msg,
-                status=400,
-            )
         except JSONDecodeError as e:
-            error_msg: ErrorMessage = {
-                "error": {
```

Comment:
why are you eliminating the typing?

Reply:
mh, not sure how it happened, I must have messed this up while trying to remove the right block, will fix it